apps/web/src/components/layout/AppSidebar.tsx

```diff
@@ -139,6 +152,96 @@ function NavLink({
   );
 }
 
+function ProfileDropdown({ isExpanded }: { isExpanded: boolean }) {
+  const router = useRouter();
+  const { state, logout } = useAuth();
+  const { user } = state;
+
+  const initials = user?.name
+    ? user.name
+        .split(" ")
+        .map((n) => n[0])
+        .join("")
+        .toUpperCase()
+        .slice(0, 2)
+    : "U";
```

Comment:
```suggestion
  const initials = user?.name
    ? user.name
        .split(" ")
        .filter((n) => n.length > 0)
        .map((n) => n[0])
        .join("")
        .toUpperCase()
        .slice(0, 2)
    : "U";
```
If the name contains extra spaces (e.g., "John  Doe"), this will create empty strings in the array, and n[0] will be undefined. We need to filter out empty strings, as above.